src/data/uma-title-requirements.csv

```diff
@@ -139,3 +139,4 @@
 "どちらもファン数が460000人以上のウマ娘から想いを継承し、\nファン数が480000人以上になる","Inherit memories from parents with ≥460000 fans\nEarn a total of ≥480000 fans"
 "全レースで2着以内で、\n菊花賞、天皇賞（春）、宝塚記念を勝利し、\n目標を全て達成し育成を完了する","<size=22>Finish ≥2nd in every single race\nWin the Kikka Sho, Tenno Sho (Spring), and Takarazuka Kinen\nClear all objectives and complete the training</size>"
 "作戦「追込」で皐月賞でラストの直線で4人以上を抜いて勝利し、\n作戦「追込」で天皇賞（春）を勝利し、\n基礎能力[根性]が1200以上になる。","<size=22>Win the Satsuki Sho as a chaser by breaking through ≥4 people in the last straight\nWin the Tenno Sho (Spring) as a chaser\nReach ≥1200 guts</size>"
+"皐月賞を3 1/2バ身差以上で勝利し、\n日本ダービーを5バ身差以上で勝利し、\n菊花賞を7バ身差以上で勝利し、有馬記念を二連覇する","<size=22>Win the Satsuki Sho, Japan Derby, and Kikka Sho with\ndifference of ≥3 1/2 lengths, ≥5 lengths, and ≥7 lengths\nand Win the Arima Kinen twice</size>"
```

Comment:
Forgot to lowercase the "Win" in "and Win the Arima Kinen twice"

Reply:
geez, i just woke up lol